cpu/esp_common/esp-wifi/esp_wifi_params.h

```diff
@@ -44,26 +44,30 @@
 /**
  * @brief   SSID of the AP to be used.
  */
-#if !defined(ESP_WIFI_SSID) && !defined(ESP_WIFI_AP_PREFIX)
+#ifndef ESP_WIFI_SSID
 #define ESP_WIFI_SSID       "RIOT_AP"
 #endif
 
-/**
- * @brief   Prefix to be used as part of the SSID (e.g.: RIOT_AP_aabbccddeeff)
- */
-#if !defined(ESP_WIFI_SSID) && !defined(ESP_WIFI_AP_PREFIX) || defined(DOXYGEN)
-#define ESP_WIFI_AP_PREFIX "RIOT_AP_"
-#endif
-
 /**
  * @brief   Passphrase used for the AP as clear text (max. 64 chars).
  */
-#ifdef DOXYGEN
+#ifndef ESP_WIFI_PASS
```

Comment:
`ESP_WIFI_PASS` was intentionally defined only when `DOXYGEN` is also defined to allow `ESP_WIFI_PASS` to be left undefined for using APs without authentication. There are no other changes in this PR affected by this change.